02-OOP-Inheritance/Ex02/script.js

```diff
@@ -0,0 +1,37 @@
+class eventEmitter {
+	constructor() {
+		this.events = [];
+	}
+	on(eventName, callback) {
+		if(this.events[eventName] != callback) {
+			callback();
```

Comment:
The on method is to attach a callback to a particular event. Not need to execute the callback here, just added to the array of callbacks for each event.